lib/cli.js

```diff
@@ -118,6 +118,10 @@ exports.run = () => {
 		throw new Error(colors.error(figures.cross) + ' Watch mode is not available in CI, as it prevents AVA from terminating.');
 	}
 
+	if (hasFlag('--concurrency') || hasFlag('-c')) {
```

Comment:
@sindresorhus I see. In that case, `((hasFlag('--concurrency') || hasFlag('-c')) && !cli.flags.concurrency)` should work good, right?